core/src/main/scala/cromwell/core/simpleton/WdlValueBuilder.scala

```diff
@@ -0,0 +1,145 @@
+package cromwell.core.simpleton
+
+import wdl4s.TaskOutput
+import wdl4s.types.{WdlArrayType, WdlMapType, WdlPrimitiveType, WdlType}
+import wdl4s.values.{WdlArray, WdlMap, WdlValue}
+
+import scala.language.postfixOps
+import cromwell.core.simpleton.WdlValueSimpleton._
+
+
+/** Builds arbitrary WdlValues from WdlValueSimpletons. */
+object WdlValueBuilder {
+
+  private val SimpletonPattern = """
```

Comment:
I just found that I neither understood what the regex was nor what it did which wasn't such a good situation. Like I said though, I'm happy to be an outlier